src/app/components/MediaLoader/index.tsx

```diff
@@ -185,6 +180,10 @@ const MediaLoader = ({
 
   if (isLite) return null;
 
+  const {
+    model: { pageIdentifierOverride },
+  } = filterForBlockType(blocks, 'mediaOverrides');
+
```

Comment:
I am actually wondering if we now override the countername within our tvMedia function like we do the language/episode title, instead of doing it here?

Reply:
Based on my understanding of how this would be achieved, I fear that this might carry over
[this](https://github.com/bbc/morph-modules/blob/master/news-av-player/src/smp.js#L197-L238) approach of building configurations which in my opinion can be quite confusing.
This is because we'd have to set the counterName in `basePlayerConfig` and then undo that
value by altering the counterName to match the override.
As such, may I suggest that we retain this as is, because here we can tell
counterName can only be 1 of 2 values i.e  pageIdentifierOverride or pageIdentifier, in comparison
to a scenario where it's set and then unset elsewhere. This might create a debugging/readability challenge.

Reply:
That's fair. I am all for keeping this as readable, understandable and debuggable as possible! Thanks for the explanation.